SafeExamBrowser.SystemComponents/VirtualMachineDetector.cs

```diff
@@ -37,27 +39,136 @@ public VirtualMachineDetector(ILogger logger, ISystemInfo systemInfo)
 			this.systemInfo = systemInfo;
 		}
 
-		public bool IsVirtualMachine()
+		private bool IsVirtualSystemInfo(string biosInfo, string manufacturer, string model)
 		{
-			var biosInfo = systemInfo.BiosInfo.ToLower();
-			var isVirtualMachine = false;
-			var macAddress = systemInfo.MacAddress;
-			var manufacturer = systemInfo.Manufacturer.ToLower();
-			var model = systemInfo.Model.ToLower();
-			var devices = systemInfo.PlugAndPlayDeviceIds;
+			bool isVirtualMachine = false;
```

Comment:
Please use `var` wherever possible and meaningful, it helps having a cleaner code view as well as enforcing proper variable naming.